modules/admin/src/messages/fr/admin.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+return [
+    /** Start::No translation **/
+    'layout_debug_assetmanager_forcecopy' => 'AssetManager forceCopy',
+    'layout_debug_transfer_exceptions' => 'Transfer Exceptions',
+    'layout_debug_yii_debug' => 'YII_DEBUG',
+    'layout_debug_yii_env' => 'YII_ENV',
+    'layout_debug_app_language' => 'Yii App Language',
+    'layout_debug_luya_language' => 'Luya Language',
+    'layout_debug_yii_timezone' => 'Yii Timezone',
+    'layout_debug_php_timezone' => 'PHP Timezone',
+    'layout_debug_php_ini_memory_limit' => 'PHP memory_limit',
+    'layout_debug_php_ini_max_exec' => 'PHP max_execution_time',
+    'layout_debug_php_ini_post_max_size' => 'PHP post_max_size',
+    'layout_debug_php_ini_upload_max_file' => 'PHP upload_max_filesize',
+    /** End::No translation **/
+
+    'language_name' => 'Anglais',
```

Comment:
It should be name of the current language, so in this case i guess it would be `Français`.